docs/source/building.rst

```diff
@@ -154,10 +154,21 @@ Regression Manager
 
 .. envvar:: TESTCASE
 
+    A comma-separated list of tests to run.
+    Does an exact match on the test name.
+
+    .. versionchanged:: 2.0
+
+        Previously, if one than one test matched a test name in the :envvar:`TESTCASE` list,
+        only the first test that matched that test name in the :envvar:`MODULE` list was run.
+        Now, all tests that match the test name across all :envvar:`MODULE`\ s are run.
+
+.. envvar:: COCOTB_TEST_FILTER
```

Comment:
This section deserves a note or warning that only one of `TESTCASE`/`COCOTB_TEST_FILTER` can be used.